### PR TITLE
Green dev

### PR DESCRIPTION
**Pull request by philjlharvey**

### CODE REVIEW

**Review by gardnersam — APPROVED** [review]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -193,6 +194,66 @@ steps:
...
+      from_secret: DEV_WORKFLOW_SERVICE_GREEN_NAME
+    WORKFLOW_SERVICE_URL:
+      from_secret: DEV_COP_SERVICE_GREEN_URL
+  when:
+    branch:
+    - green-dev
+    event:
+    - promote
+    target:
+    - dev_cop
```
Comment (rohitsamala) [inline .drone.yml:255]: target name inline with UI codeas well change it to green_dev

File: .drone.yml [hunk 2]
```diff
@@ -193,6 +194,66 @@ steps:
...
+
+platform:
+  os: linux
+  arch: amd64
+
+workspace:
+  path: /build/src
+
+steps:
+- name: deploy_to_dev_cop_service
```
Comment (rohitsamala) [inline .drone.yml:210]: deploy_to_green_cop_service

File: .drone.yml [hunk 3]
```diff
@@ -851,6 +912,7 @@ steps:
   when:
     branch:
     - master
+    - green-dev
```
Comment (rohitsamala) [inline .drone.yml:915]: remove green-dev as we only deploy from master

**Comment by philjlharvey** [comment]
recreate after signing all cmmits

**Pull request closed without merging**